web/create_website.sh

```diff
@@ -0,0 +1,37 @@
+
+# we start by cloning the repository that contains the website
+git clone https://github.com/cernvm/web-team
```

Comment:
This part we can leave to the Jenkins SCM plugin, I think

Reply:
True, but we couple the script and the jenkins. I usually prefer to have a script that we can run everywhere.